rocketc/rocketc.py

```diff
@@ -43,22 +46,34 @@ class RocketChatXBlock(XBlock, XBlockWithSettingsMixin, StudioEditableXBlockMixi
     )
 
     default_channel = String(
-        display_name="Default Channel",
-        default="",
+        display_name="Specific Channel",
         scope=Scope.content,
         help="This field allows to select the channel that would be accesible in the unit",
         values_provider=lambda self: self.get_groups(),
     )
-    default_group_enable = Boolean(
+
+    ui_is_block = Boolean(
         default=False,
         scope=Scope.user_state,
         help="This is the flag for the initial channel",
     )
 
+    channel = String(
```

Comment:
I'm not sure if channel is the best name for this setting because this is not a channel name, it's rather the channel type, what do you think?

Reply:
selected_view ???